libnavigation-base/src/main/java/com/mapbox/navigation/base/route/NavigationRoute.kt

```diff
@@ -151,19 +155,29 @@ class NavigationRoute internal constructor(
             routerOrigin: RouterOrigin,
             routeParser: SDKRouteParser = NativeRouteParserWrapper
         ): List<NavigationRoute> {
+            logI("NavigationRoute.create called for $routeRequestUrl", TAG)
             return coroutineScope {
                 val deferredResponseParsing = async(ThreadController.DefaultDispatcher) {
-                    DirectionsResponse.fromJson(directionsResponseJson)
+                    logI("parsing directions response to java model from $routeRequestUrl", TAG)
+                    val result = DirectionsResponse.fromJson(directionsResponseJson)
+                    logI("parsed directions response to java model from $routeRequestUrl", TAG)
+                    result
                 }
                 val deferredNativeParsing = async(ThreadController.DefaultDispatcher) {
-                    routeParser.parseDirectionsResponse(
+                    logI("parsing directions response to RouteInterface from $routeRequestUrl", TAG)
+                    val result = routeParser.parseDirectionsResponse(
```

Comment:
There's a `withContext` call in `parseDirectionsResponse`. Maybe also add logs there?

Reply:
I see it's already logged under method `parseDirectionsResponse` 🤔

Reply:
`DirectionResponseUtils`
We come there from `RouterWrapper#getRoute`.

Reply:
does https://github.com/mapbox/mapbox-navigation-android/pull/5965#discussion_r907103915 
 about `parseDirectionsResponse.fold { error -> /**is not logged */ }`? @dzinad

Reply:
> `DirectionResponseUtils` We come there from `RouterWrapper#getRoute`.

I see. `NavigationRoute.createAsync` is called right after switching the context in `parseDirectionsResponse`. `NavigationRoute.createAsync` logs call at the beginning, so I think it's not necessary to add one more log just before it.

Reply:
> I see. NavigationRoute.createAsync is called right after switching the context in parseDirectionsResponse. NavigationRoute.createAsync logs call at the beginning, so I think it's not necessary to add one more log just before it.

Yes. I was talking about that. OK, no need, agreed.